docs/workflows/phylogenetic_construction/clair3_variants.md

```diff
@@ -0,0 +1,134 @@
+
+
+PHB `Clair3_Variants` supports 10 model configuration for `clair3` ONT:
+
+| Model | Chemistry | Source |
+|-------|-----------|---------|
+| `r941_prom_sup_g5014` | R9.4.1 | Clair3 1.0.10 Release |
+| `r941_prom_hac_g360+g422` | R9.4.1 | Clair3 1.0.10 Release |
+| `r941_prom_hac_g238` | R9.4.1 | Clair3 1.0.10 Release |
+| `r1041_e82_400bps_sup_v500` | R10.4.1 | [nanoporetech/rerio](https://github.com/nanoporetech/rerio?tab=readme-ov-file#clair3-models) |
+| `r1041_e82_400bps_hac_v500` | R10.4.1 | [nanoporetech/rerio](https://github.com/nanoporetech/rerio?tab=readme-ov-file#clair3-models) |
+| `r1041_e82_400bps_sup_v410` | R10.4.1 | [nanoporetech/rerio](https://github.com/nanoporetech/rerio?tab=readme-ov-file#clair3-models) |
+| `r1041_e82_400bps_hac_v410` | R10.4.1 | [nanoporetech/rerio](https://github.com/nanoporetech/rerio?tab=readme-ov-file#clair3-models) |
+| `ont` | Various | Legacy (Recommended for Guppy3 and Guppy4) |
+| `ont_guppy2` | Various | Legacy (For Guppy2 data) |
+| `ont_guppy5` | Various | Legacy (For Guppy5 data) |
+
+### Inputs
+
+!!! warning "Note on Haploid Settings"
+   Several parameters are set by default for haploid genome analysis:
+
+   - clair3_disable_phasing is set to `true` since phasing is not relevant for haploid genomes
+   - clair3_include_all_contigs is set to `true` to ensure complete genome coverage
+   - clair3_enable_haploid_precise is set to `true` to only consider homozygous variants (1/1), which is appropriate for haploid genomes
+
+<div class="searchable-table" markdown="1">
+
+| **Terra Task Name** | **Variable** | **Type** | **Description** | **Default Value** | **Terra Status** |
+|---|---|---|---|---|---|
+| clair3_variants_ont | **clair3_cpu** | Int | Number of CPUs to use | 4 | Optional |
+| clair3_variants_ont | **clair3_disable_phasing** | Boolean | Disable whatshap phasing | true | Optional |
+| clair3_variants_ont | **clair3_disk_size** | Int | Disk size in GB | 100 | Optional |
+| clair3_variants_ont | **clair3_docker** | String | Docker container for task | us-docker.pkg.dev/general-theiagen/staphb/clair3:1.0.10 | Optional |
+| clair3_variants_ont | **clair3_enable_gvcf** | Boolean | Output gVCF format | false | Optional |
+| clair3_variants_ont | **clair3_enable_haploid_precise** | Boolean | Enable haploid precise calling, only 1/1 is considered as a variant | true | Optional |
+| clair3_variants_ont | **clair3_enable_long_indel** | Boolean | Enable long indel calling | false | Optional |
+| clair3_variants_ont | **clair3_include_all_contigs** | Boolean | Call variants on all contigs, should always be true for non-human samples | true | Optional |
+| clair3_variants_ont | **clair3_memory** | Int | Memory allocation in GB | 8 | Optional |
+| clair3_variants_ont | **clair3_model** | String | Model name for variant calling, see latest models: https://github.com/nanoporetech/rerio?tab=readme-ov-file#clair3-models | r941_prom_hac_g360+g422 | Optional |
```

Comment:
Instead of linking to nanopore documentation, why not link to the table that you have above with the models already included in the container?